packages/auth0-auth-js/EXAMPLES.md

```diff
@@ -314,6 +314,20 @@ const tokenResponse = await authClient.backchannelAuthentication({
 > Using Client-Initiated Backchannel Authentication requires the feature to be enabled in the Auth0 dashboard.
 > Read [the Auth0 docs](https://auth0.com/docs/get-started/authentication-and-authorization-flow/client-initiated-backchannel-authentication-flow) to learn more about Client-Initiated Backchannel Authentication.
 
+By default, the `backchannelAuthentication` method will handle the entire flow, including polling the token endpoint until the user has completed the authentication on their device. If you want to handle the polling yourself, you can do so by calling `initiateBackchannelAuthentication` and `backchannelAuthenticationGrant` separately:
+
+```ts
+const { authReqId, expiresIn, interval } = await authClient.initiateBackchannelAuthentication({
+  bindingMessage: '',
+  loginHint: {
+    sub: 'auth0|123456789'
+  }
+});
+
+// Poll the token endpoint using the authReqId
+const tokenResponse = await authClient.backchannelAuthenticationGrant({ authReqId });
+```
+
```

Comment:
Can we add something in the md file here that explains what interval should be used for ? and what expiresIn does ?